src/App.js

```diff
@@ -1,25 +1,93 @@
-import logo from './logo.svg';
-import './App.css';
+import Sidebar from "./components/Sidebar"
+import Editor from "./components/Editor"
+import Split from "react-split"
+import {nanoid} from "nanoid"
+import { useEffect, useState } from "react"
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
 
-export default App;
+export default function App() {
+    const [notes, setNotes] = useState(() => JSON.parse(localStorage.getItem("notes") ) || [])
+    const [currentNoteId, setCurrentNoteId] = useState(
+        (notes[0] && notes[0].id) || ""
```

Comment:
to improve your code readability, you can make this a ternary operator 
```suggestion
        notes[0] ? notes[0].id : ""
```